frontend/src/layout/navigation-3000/sidepanel/panels/activation/activationLogic.ts

```diff
@@ -276,16 +258,6 @@ export const activationLogic = kea<activationLogicType>([
                                 url: 'https://posthog.com/tutorials/event-tracking-guide#setting-up-custom-events',
                             })
                             break
-                        case ActivationTasks.InstallFirstApp:
-                            tasks.push({
-                                id: ActivationTasks.InstallFirstApp,
-                                name: 'Install your first app',
-                                description: `Extend PostHog's core functionality with apps`,
-                                completed: installedPlugins.length > 0,
```

Comment:
we can re-add this in some format later, but this was always true in the past, because we by default enable geoIP for everyone